src/wasm/wasm-emscripten.cpp

```diff
@@ -908,6 +970,15 @@ Literal AsmConstWalker::tableIndexForName(Name name) {
   return tableIndices[name] = Literal(tableOffsets[0]++);
 }
 
+Name AsmConstWalker::globalImportForName(Name name) {
+  if (globalImports.count(name)) {
+    return globalImports[name];
+  }
+  std::string result = "gimport$";
+  result += name.c_str();
+  return globalImports[name] = Name(result.c_str());
```

Comment:
This might just be me but I pretty much never like assignment-expressions.
```suggestion
  auto ret = Name(result.c_str());
  globalImports[name] = ret;
  return ret;
```

Reply:
Applied change.